students/jjakub/session03/mailroom.py

```diff
@@ -0,0 +1,67 @@
+
+import sys
+
+user_menu = "\n".join(("Would you like to: ",
+"1 - Send a Thank You",
+"2 - Create a Report",
+"3 - Quit",
+">>>> "))
+
+donor_db = [
+("Alex", [53898.23, 2653.00, 105.07]) ,
+("Jeff", [183.00, 200.00]) ,
+("Paul", [33941.98]) ,
+("Mark", [1.65, 55.00]) ,
+("John", [2705.71])]
+
+def sort_key(donor_db):
+    return sum(donor_db[1])
+
+def send_thanks():
+    name = ''
+    while not name or name.lower() == 'list':
+        name = input("Please enter the full name: ")
+        if name.lower() == 'list':
+            print(' '.join(x[0] for x in donor_db))
+
+    donation = int(input("Please enter the donation amount: "))
+    for i in range(len(donor_db)):
+        if name.lower() ==  donor_db[i][0].lower():
+            donor_db[i][1].append(donation)
+            return
```

Comment:
Hmm, if you return from here, then what about the thank you letter for the found donor?

I suppose you may want a logic like the following, using Python's unique `else` block for `for`?:
```
    for i in range(len(donor_db)):
        if name.lower() ==  donor_db[i][0].lower():
            donor_db[i][1].append(donation)
            return
    else:
        donor_db.append((name, [donation]))
```
Of course we can avoid all this complication next week with `dict`-based donor DB...